examples/experimental/i3s-17-and-debug/bounding-volume-layer.js

```diff
@@ -32,6 +31,16 @@ export default class BoundingVolumeLayer extends CompositeLayer {
     };
   }
 
+  updateState({changeFlags}) {
+    if (changeFlags.propsChanged) {
+      const {layerMap} = this.state;
+
+      for (const key in layerMap) {
```

Comment:
This will update all layers on any prop change. Safe but perhaps some unnecessary work. Just watch out for perf issues.

Reply:
Yes. I tried to do update only if `boundingVolumeType` is changed. And it works better but we also provide a callback which is responsible for `boundingVolume` coloring. And if we change coloring mode in the debug app bounding volumes colors don't change. I didn't find a way to handle this. Maybe we can provide a coloring mode prop to the `boundingVolume` layer. If you remember in the debug app we have (random by tile, random by depth, user custom choice). But this prop will be useless for bounding-volume-layer just for checking is the `updateState` function.

Reply:
OK.